### PR TITLE
LPS-184638

### PR DESCRIPTION
**Pull request by ces-quanhuynh**
Hi team,
Please help me to review this PR
Thank you.
Quan

# Motivation
[LPS-184638](https://liferay.atlassian.net/browse/LPS-184638) [FE] Add a checkbox to set/unset the propagation of permissions when creating a new folder

# Solution
LPS-184643
Create a new attribute called defaultValue in InputPermissionTag and set it true. It will be set true when creating new DLFolder

cc @locpham97 @ces-luannguyen @tototrinh

### CODE REVIEW

File: util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java [hunk 1]
```diff
@@ -36,18 +36,20 @@ public static String doTag(
...
-			false, pageContext);
+			false, false, pageContext);
 	}
 
 	public static String doTag(
 			String page, String formName, String modelName,
-			String permissionPropagationCheckboxLabel, boolean reverse,
-			PageContext pageContext)
+			String permissionPropagationCheckboxLabel, boolean defaultValue,
+			boolean reverse, PageContext pageContext)
```
Comment (tototrinh) [inline util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java:45]: The parameters should be sorted alphabetically.

File: util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java [hunk 2]
```diff
@@ -36,18 +36,20 @@ public static String doTag(
 
 		return doTag(
 			_PAGE, formName, modelName, permissionPropagationCheckboxLabel,
-			false, pageContext);
+			false, false, pageContext);
```
Comment (tototrinh) [inline util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java:39]: Based on the attribute's description
`<description>Whether the checkbox is selected by default. The default value is <![CDATA[<code>true</code>]]>.`
The default value should be true, right?

File: util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java (hunk 1 above)
Comment (ces-quanhuynh) [reply util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java]: Fixed it. Thanks

File: util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java (hunk 2 above)
Comment (ces-quanhuynh) [reply util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java]: Fixed it. Thanks

**Comment by ces-quanhuynh** [comment]
Hi @locpham97 @tototrinh 
I've updated the PR following Ms To's recent review. 
Please take a look.
Thanks

File: util-taglib/src/META-INF/liferay-ui.tld [hunk 3]
```diff
@@ -1304,6 +1304,13 @@
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<description>Whether the checkbox is selected by default. The default value is <![CDATA[<code>true</code>]]>.</description>
```
Comment (locpham97) [inline util-taglib/src/META-INF/liferay-ui.tld:1308]: I think it should be more clarify in here. Maybe it is should "Whether the permissionPropagationCheckbox is selected by default."

Comment (ces-quanhuynh) [reply util-taglib/src/META-INF/liferay-ui.tld]: Fixed it. Thanks

File: util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java [hunk 4]
```diff
@@ -102,6 +107,10 @@ public String getPermissionPropagationCheckboxLabel() {
 		return _permissionPropagationCheckboxLabel;
 	}
 
+	public boolean isPermissionPropagationCheckboxValueChecked() {
```
Comment (ces-luannguyen) [inline util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java:110]: I think isPermissionPropagationCheckboxValue is enough, just like other naming convention.

Comment (ces-quanhuynh) [reply util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java]: Fixed it. Thanks

**Review by ces-luannguyen — APPROVED** [review]

**Review by ces-totrinh — APPROVED** [review]
LGTM. Thanks.

**Comment by ces-totrinh** [comment]
@hhtank this PR passed our review. Please help take a look at this. Many thanks.

**Comment by hhtank** [comment]
@tototrinh LGTM, thanks.

**Comment by ces-totrinh** [comment]
Merged. Thanks all.

**Pull request merged**